Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -748,8 +748,12 @@ static String mutationObserverNotificationString()
 
         // FIXME: Need to apply a rotation transform here for correct rendering in rotated pages.
 
-        if (haveSelection)
+        if (haveSelection) {
+            AffineTransform transformForBox { [page transformForBox:kPDFDisplayBoxCropBox] };
```

Comment:
Should we get this transform from `PDFDocumentLayout` (akin to `PDFDocumentLayout::toPageTransform()`) instead?

Reply:
Indeed this is just the inverse of `toPageTransform()`.

Reply:
Using that instead in https://github.com/WebKit/WebKit/commit/69e524535d647f780bea1c444e203a1eb60c1dba